### PR TITLE
Color changed to gold, navbar rendered, buttons re-styled

### PR DESCRIPTION
**Pull request by dsryan8197**

### CODE REVIEW

**Review by a4heng — APPROVED** [review]

**Review by trentcurrie — COMMENTED** [review]
discussed changes as group, approved after resolving all merge conflicts

**Review by trentcurrie — APPROVED** [review]

**Pull request merged**
